sites/public/styles/overrides.scss

```diff
@@ -85,9 +97,13 @@
             text-transform: none;
 
             @screen md {
-              @apply text-2xl;
+              @apply text-lg;
               margin: 0;
             }
+
+            @screen xl {
+              @apply text-2xl;
```

Comment:
Do we still want the margin: 0 here?

Reply:
The margin: 0 still applies from the @screen md so we don't need to add it here. `@screen md` translates to `@media (min-width: 768px)` which means it applies to all screens above 768px. The way to think of it is you design for mobile first and then add whatever you tweaks you need as it gets bigger.

Reply:
Ah, thanks for the clarification! I still have much to learn, it seems. :D